bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/shellyGen2_sensor.xml

```diff
@@ -38,6 +38,7 @@
 		<category>Sensor</category>
 		<channel-groups>
 			<channel-group id="sensors" typeId="sensorData"/>
+			<channel-group id="relay" typeId="relayChannel"/>
```

Comment:
New group of channels so we should find upgrade instructions.

Reply:
No, the framework handles this situation and dynamically adds the group information.
In fact the channel were already created dynamically, but the missing declaration of the group resulted in a missing group label on UI level. Adding the info brings in the correct label and doesn't require further upgrade information.

Reply:
Ok